Dockerfile.aro-multistage-onebranch

```diff
@@ -1,22 +0,0 @@
-# Uses a multi-stage container build to build the RP in OneBranch.
```

Comment:
can you remind me again why we are deleting this

Reply:
I left a comment in our thread above hopefully clarifying this a bit.